### DISCUSSION TITLE
How to implement a function includes "if and else"

### DISCUSSION THREAD

**Discussion by sSajjad90** [idaholab/moose#17072] [Q&A Modules: General] [answered]
Hi everyone,
I am going to implement a "ParsedMaterial", which function includes three parts via time. In this regard, I need an "if" function with two conditions (an "if" and two else) like below, for example. But its syntax is not correct. Would you please let me know what the right way to write this command is?

  [./a1]
    type = ParsedMaterial
    function = 'if(t<=15, ac*t && 15<t<20, ac*t*t,0)'
    material_property_names = 'ac t'
    f_name = a1
  [../]

I should mention that it works when there is a condition as follows, but I can't define it for the state with two conditions.

  [./a1]
    type = ParsedMaterial
    function = 'if(t<15, ac*t ,0)'
    material_property_names = 'ac t'
    f_name = a1
  [../]

Thanks in advance

**Comment by WilkAndy** [comment, marked as answer]
I'm not sure i understand what you want, but how about
```
if(t<=15, low_value, if(t<=20, medium_value, high_value))
```

**Reply by sSajjad90** [reply]
Many thanks, It works now.
